tasks/install_odl.yml

```diff
@@ -1,5 +1,23 @@
 ---
+- set_fact:
+    os="{{ ansible_distribution }}_{{ ansible_distribution_major_version }}.0"
```

Comment:
Some inline docs would be good here. For example, what ansible_distribution_version might typically resolve to. Also not clear without digging a bit why we need these facts. Also would be nice to doc the strangeness with x prefix to Ubuntu, and .0 suffix for Debian. Otherwise, I don't see a more clear/elegant way of doing this logic.